### ISSUE TITLE
Update to browserslist v2

### ISSUE THREAD

**Issue by ismay** [anandthakker/doiuse#68]
Was just released: https://github.com/ai/browserslist/blob/master/CHANGELOG.md#20. The currently used version (1.1.1) is quite outdated.

**Comment by Kovensky** [comment]
The browserslist required by doiuse is old enough that it cannot be deduplicated by npm3+/yarn and it crashes on newer versions of the caniuse database.

```
BrowserslistError: Unknown browser baidu
    at error (.../node_modules/doiuse/node_modules/browserslist/index.js:37:11)
    at Function.browserslist.checkName (.../node_modules/doiuse/node_modules/browserslist/index.js:320:18)
    at Function.select (.../node_modules/doiuse/node_modules/browserslist/index.js:591:38)
    at .../node_modules/doiuse/node_modules/browserslist/index.js:207:41
    at Array.forEach (native)
    at browserslist (.../node_modules/doiuse/node_modules/browserslist/index.js:196:13)
    at new BrowserSelection (.../node_modules/doiuse/lib/browsers.js:17:18)
    at missing (.../node_modules/doiuse/lib/missing-support.js:76:18)
    at doiuse (.../node_modules/doiuse/lib/doiuse.js:42:25)
```

**Comment by ai** [comment]
Fixed in 3.0
